autopush/tests/test_websocket.py

```diff
@@ -54,6 +54,7 @@ def setUp(self):
         self.proto.sendClose = self.close_mock = Mock()
         self.proto.transport = self.transport_mock = Mock()
         self.proto.closeHandshakeTimeout = 0
+        self.proto.autoPingInterval = 300
```

Comment:
Do we want to make this a config value now? I'm  being a bit paranoid because the value is a guess and having it a config would let us make a tweak without pushing code.